templates/isb_cgc/citations.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'base.html' %}
+{% block content %}
+    <div class="container">
+        <div class="row">
+            <div id="citations" class="col-lg-12">
+                <h2>Citations</h2>
+                <ol type="1">
+                    {% for paper in papers %}
+                        <li>
+                            {{ paper.authors }}
+                            <a href="https://www.mendeley.com/catalogue/{{ paper.id }}/">
+                                {{ paper.title }}. </a>
```

Comment:
the {{ paper.title }} seems to include some html tabs which are not properly escaped.
For example Citation No.4 displays <i> tags in the text.

Reply:
Oh I see. I'll look into that.

Reply:
Resolved.